R/archivePythonEnv.R

```diff
@@ -0,0 +1,64 @@
+#' writePythonVirtualEnvLockFile
+#'
+#' Write Python virtual environment lock file specifying package versions currently installed in the given
+#' virtual environment. Using the lock file, a new virtual environment can be built containing exactly the same
+#' package versions.
+#'
+#' @param outputDir Path to the directory where the lock file will be written
+#' @param pythonConfig as produced by \code{\link{checkPythonEnv}}. Defaults to NULL. If not provided, the
+#'  RETICULATE_PYTHON environment variable is used
+#'
+#' @author Mika Pflüger, Tonn Rüter
+#' @importFrom reticulate py_config
+#' @importFrom yaml as.yaml yaml.load
+#' @importFrom utils head
+#' @export
+archivePythonEnv <- function(outputDir, pythonConfig = reticulate::py_config()) {
+  if (!dir.exists(outputDir)) {
+    stop("Provided path '", outputDir, "' does not exist.")
+  }
+  dateAndTime <- format(Sys.time(), "%y%m%d_%H%M%S")
```

Comment:
```suggestion
  dateAndTime <- format(Sys.time(), "%y%m%dT%H%M%S")
```
I think this would be more in line with [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), but take it or leave it, it's up to personal preference.

Reply:
I'll take it :)